### DISCUSSION TITLE
Is there a place where users of HO can disscuss about it?

### DISCUSSION THREAD

**Discussion by theobserver** [ho-dev/HattrickOrganizer#1504] [General]
Hi all,
i would like to ask if there is a place youtube channel, site or something similar where users of hattrick organizer can discuss about it and how to use it with tutorials e.t.c.

Thw work of akasolace updating HO is great and it would be great if there is/was a community of the people using it trying to help each other with tutorials or faq e.t.c. 

I, for example, while i use it, i have many questions and i cannot find relevant documentation  except some scattered info usually for the old versions.

I could setup a quick forum but i have not the time to create documentation. A community could help on this.

Thank you for your time reading this.

Nik.

**Comment by akasolace** [comment]
Hello Nik, 

There is no such thing as of yet and it sounds a good idea.  Would you feel launching such initiative? 

Best

**Comment by theobserver** [comment]
Hi akasolace,

i always admired your work with hattrick organizer and i believe you are doing much.

For the above reason, alhough at the moment i am rather busy with various family issues, i will try to find some time to setup a quick forum.

I will let you know.

Keep up the good work.

Regards,
Nik.
________________________________

**Comment by akasolace** [comment]
Thank you, Nik, for the kind work and for your intention to support the project. 
For your information, I would like to add that for almost a year now, @wsbrenk has been the one driving the project !

**Reply by tychobrailleur** [reply]
Yup, @wsbrenk deserves a statue along yours in the HT world!
